kalite/topic_tools/annotate.py

```diff
@@ -0,0 +1,155 @@
+import os
+import json
+
+from django.conf import settings as django_settings
+logging = django_settings.LOG
+
+from django.utils.translation import gettext as _
+
+from kalite import i18n
+
+from . import settings
+from .base import get_assessment_item_data
+
+
+def is_content_on_disk(content_id, format="mp4", content_path=None):
+    content_path = content_path or django_settings.CONTENT_ROOT
+    content_file = os.path.join(content_path, content_id + ".%s" % format)
+    return os.path.isfile(content_file)
+
+
+def create_thumbnail_url(thumbnail):
+    if is_content_on_disk(thumbnail, "png"):
+        return django_settings.CONTENT_URL + thumbnail + ".png"
+    elif is_content_on_disk(thumbnail, "jpg"):
+        return django_settings.CONTENT_URL + thumbnail + ".jpg"
+    return None
+
+
+def update_content_availability(content_list, language="en"):
```

Comment:
Essentially, this is a concatenation of the annotation code from get_exercise_cache and get_content_cache, properly ```if```ed.